Android/Google.Android.DataTransport/source/Xamarin.Google.Android.DataTransport.TransportApi/Xamarin.Google.Android.DataTransport.TransportApi.csproj

```diff
@@ -0,0 +1,51 @@
+<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
+  <PropertyGroup>
+    <AssemblyName>Xamarin.Google.Android.DataTransport.TransportApi</AssemblyName>
+
+    <MonoAndroidResourcePrefix>Resources</MonoAndroidResourcePrefix>
+    <MonoAndroidAssetsPrefix>Assets</MonoAndroidAssetsPrefix>
+    <AndroidUseLatestPlatformSdk>False</AndroidUseLatestPlatformSdk>
+    <AndroidUseIntermediateDesignerFile>True</AndroidUseIntermediateDesignerFile>
+    <AndroidResgenFile>Resources\Resource.designer.cs</AndroidResgenFile>
+
+    <TargetFrameworks>MonoAndroid90</TargetFrameworks>
+    <AndroidCodegenTarget>XAJavaInterop1</AndroidCodegenTarget>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <IsBindingProject>true</IsBindingProject>
+    <AndroidClassParser>class-parse</AndroidClassParser>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <PackageId>Xamarin.Google.Android.DataTransport.TransportApi</PackageId>
+    <PackageVersion>2.0.0</PackageVersion>
+    <Title>Xamarin.Google.Android.DataTransport.TransportApi</Title>
+    <PackageDescription>Bindings for Google.Android.DataTransport.TransportApi package</PackageDescription>
+    <Owners>Microsoft</Owners>
+    <Authors>Microsoft</Authors>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <PackageLicenseUrl>https://go.microsoft.com/fwlink/?linkid=2109280</PackageLicenseUrl>
+    <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=2109279</PackageProjectUrl>
+    <RequireLicenceAcceptance>true</RequireLicenceAcceptance>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <PackageTags>xamarin, android, bindings, google, datatransport, transport, api </PackageTags>
+    <PackOnBuild>true</PackOnBuild>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Folder Include="Additions\" />
+    <Folder Include="Jars\" />
+  </ItemGroup>
+  <ItemGroup>
+    <LibraryProjectZip Include="..\..\externals\android\transport-api-2.0.0.aar">
+      <Link>Jars\transport-api-2.0.0.aar</Link>
+    </LibraryProjectZip>
+  </ItemGroup>
+  <ItemGroup>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="Additions\Xamarin.Io.OpenCensus.Stats.ViewData.cs" />
```

Comment:
Nuke this, and the empty item groups in all the .csproj files.

Reply:
Yes. I forgot that when I was copying.